### ISSUE TITLE
Level: "Tomb Raider"

### ISSUE THREAD

**Issue by colonelfazackerley** [andybalaam/rabbit-escape#72]
```
:name=Tomb Raider
:description=Reach the burial chamber
:num_to_save=3
:num_rabbits=3
:dig=5
:bash=5
:block=2
:bridge=4
:rabbit_delay=10
          Q          
                     
          /\         
         /##\        
        /####\       
       /######\      
      /########\     
     /##########\    
    /####O#######\   
  ################## 
```
It's easy.

**Comment by tttppp** [comment]
I would say medium rather than easy. I managed to lose all my rabbits on the first go!

**Comment by colonelfazackerley** [comment]
I bet it didn't take you 20-60 minutes to complete, though...

**Comment by andybalaam** [comment]
Took me a surprising number of tries - probably 6. Definitely fun.

**Comment by colonelfazackerley** [comment]
merged to master f96dd5696bc0d3c0f44b4856ba75e3a7d2bc201a
